data/analysis-datasets-versions.json

```diff
@@ -1,28 +1,28 @@
 {
-  "ANNUAL_ADM0_CHANGE": "v20240404",
-  "ANNUAL_ADM0_SUMMARY": "v20240404",
-  "ANNUAL_ADM0_WHITELIST": "v20240404",
-  "ANNUAL_ADM1_CHANGE": "v20240404",
-  "ANNUAL_ADM1_SUMMARY": "v20240404",
-  "ANNUAL_ADM1_WHITELIST": "v20240404",
-  "ANNUAL_ADM2_CHANGE": "v20240404",
-  "ANNUAL_ADM2_SUMMARY": "v20240404",
-  "ANNUAL_ADM2_WHITELIST": "v20240404",
+  "ANNUAL_ADM0_CHANGE": "v20240815",
+  "ANNUAL_ADM0_SUMMARY": "v20240815",
```

Comment:
awesome, I will be able to use this version for my new widget as well 👍

Reply:
This update is a cherry pick from #4869. It's the epic for your widget.